src/utils/sendTaskUpdates.ts

```diff
@@ -5,9 +5,17 @@ export async function sendTaskUpdate(
   completed: string,
   planned: string,
   blockers: string,
+  discordId: string,
+  taskId: string,
   env: env
 ): Promise<void> {
-  const formattedString = `**Completed**: ${completed}\n\n**Planned**: ${planned}\n\n**Blockers**: ${blockers}`;
+  const taskUrl = "https://status.realdevsquad.com/tasks/" + taskId;
+  const formattedString =
+    `<@${discordId}> added an update to his task: <${taskUrl}>\n` +
```

Comment:
We should use a gender-neutral message. Like `They added an update to their task`.

Reply:
Done please check